units/INU0001/INU0001_unit.bp

```diff
@@ -610,7 +610,7 @@ UnitBlueprint {
             Name = '<LOC NomadsACUEnh_PowerArmor_Help>Powered Armor',
             Prerequisite = 'RapidRepair',
             RepairDelay = 10,
-            RepairRate = 300,
+            RepairRate = 250, --this actually means nothing just yet
```

Comment:
comments like this are not helpful. Include the change that uses this value in the same commit. These comments are not a "todo list"